### ISSUE TITLE
Unasked for Double Cursor

### ISSUE THREAD

**Issue by Butterwell** [adobe/brackets#12383]
At fairly random times, suddenly I've got a second (multiple) cursor at what seems like a random, but close location to the other cursor. I'm not holding down a cmd or ctrl or alt key. I'm not selecting text on purpose (there might be a touchpad interaction).

**Comment by redmunds** [comment]
@Butterwell The are other ways to add additional selections using shortcuts:
* Cmd/Ctrl-B (Add Next Match to Selection)
* Alt-F3 (Find All and Select)

Could you be hitting one of those?

Do you have any extensions installed? If so, there could be even more ways to add additional cursors.

Otherwise, you'll need to keep an eye out for a pattern and provide a repeatable recipe to recreate problem.

**Comment by Butterwell** [comment]
@redmunds The git extension. The total of function keys I use are ctrl-z, ctrl-x, ctrl-c, ctrl-v and  ctrl-alt-g for the git extension. Does Brackets have a built-in key logger?

**Comment by redmunds** [comment]
> Does Brackets have a built-in key logger?

No. You could try using _Debug > Show Developer Tools_ and then switch to the console to see if any errors are displayed when it happens.

**Comment by Butterwell** [comment]
This looks like an a problem in CodeMirror, since I'm using CodeMirror for a project and the behavior is showing up there too.

**Comment by ForkInSpace** [comment]
I've had the similar problem with double-cursor. _Debug>reload with extentions_ fixed it for me.
